WordPress/src/main/java/org/wordpress/android/ui/comments/unified/UnifiedCommentsActivity.kt

```diff
@@ -2,17 +2,17 @@ package org.wordpress.android.ui.comments.unified
 
 import android.os.Bundle
 import android.view.MenuItem
-import kotlinx.android.synthetic.main.toolbar_main.*
-import org.wordpress.android.R
+import org.wordpress.android.databinding.UnifiedCommentActivityBinding
 import org.wordpress.android.ui.LocaleAwareActivity
 
 class UnifiedCommentsActivity : LocaleAwareActivity() {
     public override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        setContentView(R.layout.unified_comment_activity)
+        val binding = UnifiedCommentActivityBinding.inflate(layoutInflater)
```

Comment:
Question (❓): I was under the impression that the `with(...) { ... }` pattern is agreed and should be utilised whenever possible, isn't that the case Annmarie?

```
        with(UnifiedCommentActivityBinding.inflate(layoutInflater)) {
            setContentView(root)
            setSupportActionBar(toolbarMain)
        }
```

PS: The comment same applies for all the changes you did, on all the classes that you touched, for example:
- `EngagedPeopleListActivity`
- `SuggestionActivity`
- `JetpackRemoteInstallActivity`

Reply:
@ParaskP7  - We are all over the place in the code base. Hence my comment. Let's use this space to come to an agreement on which way we'd like to move forward
 
> P.S. There are a few different to implement ViewBinding, I tried to keep things consistent with the patterns already in use. I am open to changing this if we want to go a different route. Thanks.

Reply:
👋 @zwarm !

> @ParaskP7 - We are all over the place in the code base. Hence my comment. Let's use this space to come to an agreement on which way we'd like to move forward

Yes indeed, we are, I also saw your description on that, thus I asked if that is indeed the case, using the `with(...) { ... }` going forward. Isn't this what you are aware as well, irrespective of the current consistency of the code base? If so and after we agree on it, let's use that going forward. 🙏

Reply:
@ParaskP7 `SuggestionActivity` needs to stay as is. As `binding` is referenced in `onResume` and `updateEmptyView` which is called from the EventBus handler `onEventMainThread`. The others activities have been changed and committed in 7c1d0e0

Reply:
👋 @zwarm !

As discussed... 😄